flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/ParquetColumnarRowInputFormatTest.java

```diff
@@ -294,6 +294,80 @@ void testProjectionReadUnknownField(int rowGroupSize) throws IOException {
                 });
     }
 
+    @ParameterizedTest
+    @MethodSource("parameters")
```

Comment:
Out of curiosity: What do we gain by having so much test data in terms of `rowGroupSize`? Isn't the test here the same for a groupSize of 10 and 20 for instance which means we're not gaining anything from the test runs? Or am I missing something here?